docs/src/pages/[platform]/connected-components/storage/storage-browser/examples/CustomCopyView.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { createStorageBrowser } from '@aws-amplify/ui-react-storage/browser';
+import { Button, Flex, Text } from '@aws-amplify/ui-react';
+import { mockConfig } from './mockConfig';
+
+const { StorageBrowser, useView } = createStorageBrowser({
+  config: mockConfig,
+});
+
+function CopyView() {
+  const state = useView('Copy');
+
+  return (
+    <>
+      {state.tasks.map((task) => {
```

Comment:
Would it be possible to somehow mock or provide the tasks to this view? As it stands, copy and delete are pretty boring since neither have tasks